tools/test_runner.js

```diff
@@ -80,21 +79,22 @@ Runner.prototype.checkSkipModule = function() {
 
 
 Runner.prototype.run = function() {
-  if (this.attr.skip && (this.attr.skip.indexOf('all') >= 0 ||
-      this.attr.skip.indexOf(this.driver.os) >= 0)) {
+  if (skip = this.test['skip']) {
```

Comment:
1. skip is undeclared, this is against coding rules
2. assignment in if is a bad practice, please don't do this :)